recipes/irf/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+mkdir -p "${PREFIX}"/bin
+cp bin/irf308.linux.exe "${PREFIX}"/bin/irf
```

Comment:
This is not recommended.. can we help you to compile the package with our tool chain?

Reply:
Hi, I tried to use the instructions of compiling in https://github.com/Benson-Genomics-Lab/IRF in build.sh.
`cd src
make`
and specified {{ compiler('c') }} in the meta.yaml.

But the test pipeline raised error:
`gcc -c -o irf.o  irf.3.c -O2
make: gcc: No such file or directory
make: *** [Makefile:44: irf.o] Error 127`

I am grateful if you can give us some suggestions to help us compile the package, thank you!

Reply:
You should not hardcode gcc in your makefile. On Linux we might use gcc but on OSX clang. Good practice is to use $CC in your makefile as an env variable. Then the host system can set compliers as they like. You can default to gcc but make sure an env variable can overwrite your default complier

Reply:
Thanks for your help. I modified build.sh accordingly and passed the test pipeline. Please view.